index.js

```diff
@@ -1,27 +1,159 @@
-const mongoose = require('mongoose');
+// 1. IMPORTACIONES
+
+const express = require("express")
+const app = express()
+
+const hbs = require("hbs")
+
+const { connect } = require("mongoose")
+const connectDB = require("./config/db")
+
+require("dotenv").config()
+
 
-// Import of the model Recipe from './models/Recipe.model.js'
 const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
 const data = require('./data');
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 
-// Connection to the database "recipe-app"
-mongoose
-  .connect(MONGODB_URI, {
-    useCreateIndex: true,
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
-  .then(self => {
-    console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+
+
+// 2. MIDDLEWERS
+
+app.use(express.static("public"))
+
+app.set("views", __dirname + "/views");
+app.set("view engine", "hbs");
+
+hbs.registerPartials(__dirname + "/views/partials")
+
+connectDB()
+
+
+
+// 3. RUTAS
+
+// CRUD: Patron de operaciones, que nos permite manipular datos de un proyecto de software
+// (Crear, leer, actualizar, borrar)
+
+
+
+app.get("/home", (req, res) => {
+  res.render("home")
+})
+
+
+
+
+app.get("/c-recipe", async (req, res) => {
+
+  const myNewRecipe = {
+
+    title: "Taco random",
+    level: "Promaster300",
+    ingridients: ["Mole", "Tortilla", "Sal", "Carne", "Pimienta", "Chile Morron"],
+    cuisine: "Mexasian",
+    dishType: "Platillo Principal",
+    image: "https://i.ytimg.com/vi/LTffYgdWs8M/maxresdefault.jpg",
+    duration: 200,
+    creator: "Chef Javier"
+  }
+
+
+  // ITERACION.2
+  // Crea mi receta en la base de datos
+
+  const recipeCreated = await Recipe.create(myNewRecipe)
+
+  res.render("c-recipe", {
+
+    data: recipeCreated
+ 
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+})
+
+
+// ITERACION.3
+// Crea multiples recetas en la base de datos
+
+ 
+app.get("/c-recipes", async (req, res) => {
+
+  const recipesCreated = await Recipe.insertMany(data)
+  console.log(recipesCreated)
+
+  res.render("c-recipes")
+
+})  
+
+
+
+
+//  ITERACION.4
+// Find the user with id '61973da6701c132a1892eee7' and update its duration -=120 = 100 total;
+
+Recipe.findById('61973da6701c132a1892eee7')
```

Comment:
Te sugiero utilizar el metodo https://mongoosejs.com/docs/tutorials/findoneandupdate.html, ya que el findbyId por si solo no actualiza tu base, tambien cada vez que se ejecuta tu codigo se crean id-s nuevos este no coincide con mi base en donde ejecute tu codigo 👀 
```suggestion
Recipe.findOneAndUpdate(
  { title: "Rigatoni alla Genovese" },
  { duration: 100 },
  { new: true }
)
  .then((updatedRecipeDocument) => {
    console.log("UPDATED RECIPE ->", updatedRecipeDocument);
  })
  .catch((err) => console.log("An error occurred:", err));
```